### ISSUE TITLE
Adding more exchanges

### ISSUE THREAD

**Issue by aahutsal** [openbot-tech/core#24]
Through use of [CCXT](https://github.com/ccxt/ccxt).
They support like 130 exchanges currently.

What do you think @Kisgus ?

**Comment by Kisgus** [comment]
Good idea thanks @gutsal-arsen do you have experience using their API? Sounds like a big number of exchanges to maintain :) Is the code-standard something you approve @rootkc ?

**Comment by aahutsal** [comment]
@Kisgus yeah, I've used them once. They maintain exchanges by themselves.

There are 2 kinds of APIs - unified API for all exchanges and per-exchange API. We just have to make sure its satisfy your needs and if so - refactor to use CCXT.

Looks like you use Gitcoin. I think I can do that analysis and make the plan. Few components of the system will be affected besides back/front e.g. CircleCI should include pipeline testing CCXT components/Exchanges viability and displaying that data somewhere on status page/twitter/whatever you use for system health updates.

**Comment by rootkc** [comment]
We have considered ccxt @gutsal-arsen but they don't support websockets for live candles. Otherwise it would be awesome!
